BootloaderCommonPkg/Library/BuildFdtLib/X86_BuildFdtLib.c

```diff
@@ -0,0 +1,909 @@
+/** @file
```

Comment:
By convention we'd normally name this something like "BuildFdtLibX86.c" instead. There is nothing wrong with this naming but was there a reason behind it?